github-org/README.md

```diff
@@ -0,0 +1,36 @@
+# Mondoo GitHub Organization Action
+
+A GitHub Action for using Mondoo to scan a GitHub Organization for security vulnerabilities and misconfigurations. This Action can be easily used in [.github or .github-private](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile) repositories.
```

Comment:
@chris-rock is this used for both vulnerabilities and misconfigurations? I agree that we scan for misconfigurations, but do we scan for vulnerabilities in GitHub Organizations? 🤔

Reply:
Not yet :-)